airflow/executors/kubernetes_executor.py

```diff
@@ -628,6 +629,7 @@ def clear_not_launched_queued_tasks(self, session=None):
         proper support
         for State.LAUNCHED
         """
+        from airflow.models import TaskInstance
```

Comment:
What is purpose of this change?

Reply:
Import of model's Task Instance caused another cyclic import when we run full pylint serial check.